docs/configuration/single-process-config.yaml

```diff
@@ -66,3 +66,9 @@ storage:
 
   filesystem:
     directory: /tmp/cortex/chunks
+
+  delete_store:
+    store: boltdb
+
+purger:
+  object_store_type: filesystem
```

Comment:
Since we can have different schemas for different time range, we would have to infer which schema user would want to use. If we go with the active/lastest one then moving to a different store by adding a new schema would abandon old schemas. Which is why I wanted it to be explicit.

Reply:
I see, didn't realize that. Thanks!